src/structures/Message.js

```diff
@@ -258,13 +258,13 @@ class Message extends Base {
      */
     async getQuotedMessage() {
         if (!this.hasQuotedMsg) return undefined;
-
-        const quotedMsg = await this.client.pupPage.evaluate((msgId) => {
-            let msg = window.Store.Msg.get(msgId);
-            return msg.quotedMsgObj().serialize();
-        }, this.id._serialized);
-
-        return new Message(this.client, quotedMsg);
+        try {
+            const quotedMsg = await this.client.pupPage.evaluate((msgId) => {
+                let msg = window.Store.Msg.get(msgId);
+                return msg.quotedMsgObj().serialize();
+            }, this.id._serialized);
+            return new Message(this.client, quotedMsg);
+        } catch (_) { _; }
```

Comment:
Why are we catching and ignoring the error here?

Reply:
get* byId methods are affected

Reply:
I removed this. We should never ignore errors like this - if there's something specific that we know will be thrown we should handle it directly by checking the error and doing something about it.

In my tests i didn't get any error here - let me know if you're getting them, and we can look for what the cause is.